src/ecdsa/ecdh.py

```diff
@@ -97,39 +108,123 @@ def load_private_key(self, private_key):
         return self.private_key.get_verifying_key()
 
     def load_private_key_bytes(self, private_key):
+        """
+        Load private key from plain bytes string.
+        Uses current curve and checks if key length corresponds to
+        the current curve.
+        :param private_key: private key in bytes string format
+        :return: public (verifying) key from this private key
+        """
         return self.load_private_key(
             SigningKey.from_string(private_key, curve=self.curve))
 
     def load_private_key_der(self, private_key):
+        """
+        Load private key from DER string.
+        Curve is get from DER and check if it the same as current curve
```

Comment:
`Compares the curve of the DER-encoded key with the ECDH set curve, uses the former if unset`?